tests/unit/components/cylc/tree/tree.vue.spec.js

```diff
@@ -39,7 +58,7 @@ describe('Tree component', () => {
   // mount function from Vuetify docs https://vuetifyjs.com/ru/getting-started/unit-testing/
   /**
    * @param options
-   * @returns {Tree}
+   * @returns {Wrapper<Tree>}
```

Comment:
The wrapper has different attributes and functions, so the proper annotation gives the IDE better auto-complete (e.g. `findAllComponents` and displays its signature)